docs/user-guide/projects/project-lists/README.md

```diff
@@ -41,16 +41,26 @@ You can use this projects overview to **create a multi project status dashboard*
 
 > **Please note:** Project custom fields are an Enterprise add-on and will only be displayed here for Enterprise on-premises and Enterprise cloud.
 
-With the **arrow** on the right you can display the **project description**.
+Each project is displayed in a single line. For the the fields where the text is too long to be displayed completely please use the **Expand** link. 
 
-![Open a project description in the project lists view in OpenProject](arrow-project-description.png)
+
+
+![Open a project description in the project lists view in OpenProject](expand-link-project-description.png)
 
 With the horizontal **three dots** icon on the right side of the list you can open **further features**, such as [creating a new subproject](../#create-a-subproject), [project settings](../project-settings), [archiving a project](../#archive-a-project), [copying](../#copy-a-project) and [deleting a project](../#delete-a-project). Please note that you have to be a System Administrator in OpenProject to access these features. Find out how to un-archive projects [here](../#archive-a-project).
 
 ![new subproject project list](new-subproject-project-list.png)
 
 You can choose the **columns displayed by default** in the [System settings](../../../system-admin-guide/system-settings/project-system-settings) in the Administration. 
 
+You can add the columns, as well as define the order of the columns by using the **Configure view** modal. Navigate to it via the menu in the far right corner (three dots) and click **Configure view**.
+
+![Configure view of project lists in OpenProject](configure-view-project-list.png)
+
+A window will open, allowing you to manage and reorder columns. Don't forget to click **Apply** to see the changes. 
```

Comment:
I wouldn't use the term window here but dialog or modal as it is not opening a new window.

Also I would change the see to save in this sentence "...click **Apply** to see"